katsdptelstate/rdb_reader.py

```diff
@@ -36,6 +36,8 @@ def __init__(self):
         self.n_keys = 0
         # Flag that helps to disambiguate callback errors from parser errors
         self.client_busy = False
+        # _SEPARATOR_KEY loaded from the file, if any
```

Comment:
It's handled in end_rdb. The idea is that you need to know if there was no separator in the file so that you can write one into the backend storage if not. If you prefer I can initialise it to `_` and unconditionally update the backend storage, which means that loading a file with a separator will write it twice. I *think* that's harmless.

Reply:
Thanks for the explanation - that seems to make sense :)